android/src/main/java/com/capacitorjs/community/plugins/bluetoothle/Device.kt

```diff
@@ -388,9 +388,16 @@ class Device(
             return
         }
         val bytes = stringToBytes(value)
-        characteristic.value = bytes
-        characteristic.writeType = writeType
-        val result = bluetoothGatt?.writeCharacteristic(characteristic)
+
+        val result = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+            val statusCode = bluetoothGatt?.writeCharacteristic(characteristic, bytes, writeType)
+            statusCode == 0
```

Comment:
this line doesnt look right

Reply:
I think this is just turning the int return into a bool. The `0` value here is https://developer.android.com/reference/android/bluetooth/BluetoothStatusCodes#SUCCESS

Reply:
In that case i would suggest `bluetoothGatt?.writeCharacteristic(characteristic, bytes, writeType) == BluetoothStatusCodes.SUCCESS` for better readability